R/createPackageProject.R

```diff
@@ -142,13 +143,19 @@ createPackageProject <- function(name, title = NULL,
             login = external_setup$login,
             private = external_setup$private,
             protocol = external_setup$protocol,
-            ci_activation = external_setup$ci_activation
+            ci_activation = external_setup$ci_activation,
+            project_type = "package",
+            coverage = coverage
           )
         }
 
         # README
         knit_readme()
 
+        # dummy test
+        usethis::use_testthat()
+        usethis::use_test(name = "dummy", open = FALSE)
```

Comment:
Dummy may not be a word someone wants in their commit history
```suggestion
        usethis::use_test(name = "sample", open = FALSE)
```